tests/python_build/fixtures/gles3/vertex_fragment.out

```diff
@@ -30,13 +30,13 @@ protected:
 R"<!>(
 precision highp float;
 precision highp int;
-
```

Comment:
@Ivorforce pre-commit added changes to those 3 files? Should I leave them here or do something else?

Reply:
`pre-commit` probably got confused because you had edited those files (it only formats files that have been edited). You should revert these changes, since they're not related to your PR.

Reply:
```
      - id: validate-builders
        name: validate-builders
        language: python
        entry: python tests/python_build/validate_builders.py
        pass_filenames: false
        files: ^(gles3|glsl)_builders\.py$
```
so my changes to:
```
M  gles3_builders.py
M  glsl_builders.py
```
triggered: `python tests/python_build/validate_builders.py` which changed these files. I can either revert my changes to `glsl_builders` and `gles3_builders` or leave them and the changes to the `.out` files

Reply:
I'm resolving this as intended behaviour